monkestation/_maps/random_rooms.dm

```diff
@@ -0,0 +1,76 @@
+/datum/map_template/random_room/skwh_rdm158
```

Comment:
can you make the typepaths a little more descriptive of what the room is?

Also, I feel like the prefix (`skwh_rdm`) for all of the random rooms is unnecessary